src/test/resources/openSans30.fnt

```diff
@@ -0,0 +1,714 @@
+info face="Open Sans ExtraBold" size=30 bold=1 italic=0 charset="" unicode=0 stretchH=100 smooth=1 aa=1 padding=1,1,1,1 spacing=-2,-2
+common lineHeight=42 base=33 scaleW=1024 scaleH=512 pages=1 packed=0
+page id=0 file="openSans30.png"
+chars count=709
+char id=0       x=0    y=0    width=0    height=0    xoffset=-1   yoffset=0    xadvance=0    page=0    chnl=0 
+char id=13      x=0    y=0    width=0    height=0    xoffset=-1   yoffset=0    xadvance=0    page=0    chnl=0 
+char id=33      x=1009 y=215  width=10   height=24   xoffset=0    yoffset=10   xadvance=9    page=0    chnl=0 
```

Comment:
The space character is missing. I don't really know what will happen with `char id=32` (the space char) missing from the font.